index.js

```diff
@@ -1,24 +1,12 @@
-var binwrap = require("binwrap");
-var path = require("path");
+function dummy() {
+    return Promise.resolve();
+}
 
-var packageInfo = require(path.join(__dirname, "package.json"));
-var version = packageInfo.version;
-var root =
-    "https://github.com/zwilias/elm-json/releases/download/v" +
-    version +
-    "/elm-json-v" +
-    version;
-
-module.exports = binwrap({
-    dirname: __dirname,
-    binaries: ["elm-json"],
-    urls: {
-        "darwin-arm64": root + "-aarch64-apple-darwin.tar.gz",
-        "darwin-x64": root + "-x86_64-apple-darwin.tar.gz",
-        "linux-x64": root + "-x86_64-unknown-linux-musl.tar.gz",
-        "linux-arm": root + "-armv7-unknown-linux-musleabihf.tar.gz",
-        "linux-arm64": root + "-armv7-unknown-linux-musleabihf.tar.gz",
-        "win32-x64": root + "-x86_64-pc-windows-msvc.tar.gz",
-        "win32-ia32": root + "-x86_64-pc-windows-msvc.tar.gz"
```

Comment:
Just noticed that the old version supports `linux-arm` (32-bit) and `win32-ia32` as well, and that both the 32-bit and 64-bit platforms use the same binaries?

I can easily add support for those, but just wanted to check something with you first. I don’t know much about 32-bit and 64-bit stuff. I thought that you needed different binaries for them? Is it a mistake in this code, or does it actually work?

Reply:
I asked on Discord, and the linux ARM binary is 32-bit but also works on 64-bit ARM, so I added support for that.

It’s still unclear about Windows, though.

Reply:
Update regarding Windows:

Mario Rogic
I was thinking about this combo, which I have no idea how that works:
"win32-ia32": root + "-x86_64-pc-windows-msvc.tar.gz"

BendingBender
no this one doesn't make any sense for me

BendingBender
At least what I saw in the build scripts in elm-json it looks like it's really the 64-bit x86 version that's being served to 32-bit systems. As you said, it's quite likely that it won't work but if no one ever complains, then I suppose no one actually uses it.
The good thing using rust in case of elm-json is that one doesn't need a windows system to cross-compile for windows, all the builds for all the OS' are done on a linux system and there would actually be no problem to build a 32-bit x86 windows binary. Which is completely pointless if Elm itself isn't built for this architecture. 

---

Current conclusion: The PR is good as it is, no need to add code to “support” 32-bit Windows. Both the old and new package fail in different ways on 32-bit Windows.